sass/variables/_sizing.scss

```diff
@@ -83,7 +89,7 @@ $form-box-shadow-focus: $form-box-shadow, 0 0 5px rgba(darken($form-border-color
 // Input Sizes
 $form-input-font-size: $base-font-size;
 $base-input-font-size: $form-input-font-size;
-$form-input-vertical-padding: $form-input-font-size * 0.6;
+$form-input-vertical-padding: $form-input-font-size * 0.4;
```

Comment:
make sure to remove this diff before merging, I think the right value `0.6` is on both master and the header branch now